pyproject.toml

```diff
@@ -0,0 +1,39 @@
+
+[tool.poetry]
+name = "firedantic"
+version = "0.1.0"
+description = "Pydantic base model for Firestore"
+authors = ["Digital Living International Ltd"]
+license = "BSD-3-Clause"
+readme = "README.md"
+repository = "https://github.com/digitalliving/firedantic"
+packages = [
+    {include="firedantic", from="."}
+]
+
+[tool.poetry.dependencies]
+python = "^3.6"
+pydantic = "^1.6.1"
+google-cloud-firestore = "^1.9.0"
+grpcio = "^1.32.0"
+
+[tool.poetry.dev-dependencies]
+isort = "^5.5.2"
+invoke = "^1.4.1"
+mypy = "^0.782"
+flake8 = "^3.8.3"
+pytest = "^6.0.2"
+pydevd_pycharm = "^202.7319.37"
+
+[tool.isort]
```

Comment:
since part of the tools doesn't support pyproject.toml, maybe it's better to keep tooling configuration in the `setup.cfg` and leave this file only for dependencies and poetry stuff?